grails-app/conf/Config.groovy

```diff
@@ -4,6 +4,8 @@ import com.nextdoor.rollbar.RollbarLog4jAppender
 
 grails.project.groupId = "au.org.ala" // change this to alter the default package name and Maven publishing destination
 
+grails.assets.bundle = true
```

Comment:
I think this should be removed. We don't usually want bundling during development